components/brave_rewards/resources/page/components/connect_wallet_modal.tsx

```diff
@@ -15,11 +15,11 @@ import connectWalletGraphic from '../assets/connect_wallet.svg'
 
 import * as style from './connect_wallet_modal.style'
 
-function renderProviderIcon (provider: string) {
-  switch (provider) {
-    case 'bitflyer': return <BitflyerIcon />
-    case 'gemini': return <GeminiIcon />
-    case 'uphold': return <UpholdIcon />
+function renderProviderIcon (provider: ExternalWalletProvider) {
+  switch (provider.type) {
+    case 'bitflyer': return <BitflyerIcon fill={!provider.enabled ? 'gray' : undefined} />
```

Comment:
It would be nice to get rid of these fill properties altogether and let the container style them as appropriate, but I'm not sure how difficult that would be. I'll spend a few minutes today looking into it.

Reply:
I created https://github.com/brave/brave-core/pull/15438 to get rid of these props. You can cherry-pick that commit into your branch or I can merge that separately.